services/frontend-service/pkg/config/config.go

```diff
@@ -14,11 +14,30 @@ You should have received a copy of the GNU General Public License
 along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 Copyright 2021 freiheit.com*/
+/*
```

Comment:
Seems to be a duplicated section, please remove it

Reply:
probably the comment ending got modified somehow? and automation caught it. removing the entire comment (above one as well) and adding it back via check.sh should fix it

Reply:
maybe some goland prettier function that says comment ending should be in same line?

Reply:
Already deleted